__tests__/server.test.js

```diff
@@ -121,48 +122,122 @@ describe("/api", () => {
 });
 
 describe("/api/articles/:article_id/comments", () => {
-  test("GET:200 responds with array of comments for given article id", () => {
-    return request(app)
-      .get("/api/articles/1/comments")
-      .expect(200)
-      .then(({ body: { comments } }) => {
-        expect(comments.length).toBe(11);
-        expect(comments).toBeSortedBy("created_at");
-        comments.forEach((comment) => {
-          expect(comment.article_id).toBe(1);
+  describe("GET", () => {
+    test("GET:200 responds with array of comments for given article id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(11);
+          expect(comments).toBeSortedBy("created_at");
+          comments.forEach((comment) => {
+            expect(comment.article_id).toBe(1);
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            });
+          });
+        });
+    });
+    test("GET:200 responds with empty array if article exists but no comments", () => {
+      return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(0);
+          expect(comments).toMatchObject([]);
+        });
+    });
+    test("GET:404 responds with no comments if incorrect id", () => {
```

Comment:
This isn't an accurate test description - it does respond with no comments, but more importantly it responds with an error message